Duplicati/CommandLine/Help.cs

```diff
@@ -84,6 +84,8 @@ public static void PrintUsage(TextWriter outwriter, string topic, IDictionary<st
                     topic = "exclude";
                 else if (System.Environment.CommandLine.IndexOf("--include", StringComparison.InvariantCultureIgnoreCase) >= 0)
                     topic = "include";
+                else if (System.Environment.CommandLine.IndexOf("--default-filters", StringComparison.InvariantCultureIgnoreCase) >= 0)

```

Comment:
I don't think this is required as the help routine will look at options if no match is found in the help file.